### ISSUE TITLE
`be_running` is deprecated, what should I use?

### ISSUE THREAD

**Issue by lkysow** [inspec/inspec#4059]
```
WARN: DEPRECATION: The service `be_running?` matcher is deprecated. This is only allowed for compatibility with ServerSpec
```

What should I use instead?

Also, this is in all the docs so those should be changed as well.

**Comment by myshkin-uk** [comment]
Me too - deprecation messages which don't hint at how to fix them are a PITA.

**Comment by ITJamie** [comment]
Bump. what is the new descriptor going to be?

https://github.com/inspec/inspec/blob/2d528f5899a15dab844b1672aebe49a012de6935/lib/matchers/matchers.rb#L107

theres no obvious replacement in the matchers file...

**Comment by jerryaldrichiii** [comment]
I think the issue here is we have deprecated the the hard coded matcher `be_running?` in the `matchers.rb` location. There is still a `running?` matcher in the `service` resource and that will get translated to `be_running` via RSpec during runtime.

I verified removing the matcher in `matchers.rb` does not break the `service` resource. I think we should remove the deprecation warning.

**Comment by jerryaldrichiii** [comment]
I think the issue here is we have deprecated the the hard coded matcher `be_running?` in the `matchers.rb` location. There is still a `running?` matcher in the `service` resource and that will get translated to `be_running` via RSpec during runtime.

I verified removing the matcher in matchers.rb does not break the service resource. I think we should remove the deprecation warning.

TL;DR: You should use `be_running` and that message is misleading.

**Comment by kclinden** [comment]
@jerryaldrichiii So you're saying disregard the warning, and continue to use `be_running`?

**Comment by jerryaldrichiii** [comment]
I believe that is correct, @clintoncwolfe seems to be working on this and can let you know if what I said is incorrect 😄.
